test/mocha/10-api.js

```diff
@@ -91,12 +96,14 @@ describe('bedrock-account-http', function bedrockAccountHttp() {
     });
 
     it('should return 201 if there is an email', async function() {
-      const result = await api.post('/', {email: 'newuser@digitalbazaar.com'});
+      const result = await api.post('/', {
+        email: 'newuser@digitalbazaar.com', authorization
+      });
       result.status.should.equal(201);
     });
 
     it('should return 409 for accounts with the same email', async function() {
-      const email = {email: 'multiple@digitalbazaar.com'};
+      const email = {email: 'multiple@digitalbazaar.com', authorization};
```

Comment:
Oh, this should be a new test. We should have tests both with and without authorization.

Reply:
Added 2 new tests, 1 with authz and 1 without.